OpenSim/Common/Component.h

```diff
@@ -119,6 +119,48 @@ class ComponentHasNoSystem : public Exception {
     }
 };
 
+class ComponentNoConnectorFound : public Exception {
```

Comment:
Minor, but I think `NoConnectorFound` or `ConnectorNotFound` is sufficient. The previous exceptions begin with *Component* not because they are defined in `Component` but because the it is the actual thing that is not found or already in the tree.